gqcpy/src/Basis/SpinorBasis/USpinorBasis_bindings.cpp

```diff
@@ -50,6 +50,15 @@ void bindUSpinorBasis(py::module& module) {
 
         // PUBLIC METHODS
 
+        .def(
+            "calculateAtomicSpinZ",
+            [](const GQCP::USpinorBasis<double, GQCP::GTOShell>& spinor_basis, const std::vector<size_t>& ao_list, const GQCP::Spin sigma) {
+                return spinor_basis.calculateAtomicSpinZ(ao_list, sigma);
+            },
+            py::arg("ao_list"),
+            py::arg("sigma"),
+            "Return the atomic spin-z operator for a given set of AO indices.")
```

Comment:
While these are indeed the current Python bindings, these should be updated in #452.

Reply:
As with the spin density: should I keep this for now, since it can allow us to perform some calculations alrready, or would you prefer I delete this binding for now?

Reply:
Yes, you can keep them here.